ts-web-component/src/setup-replicache.ts

```diff
@@ -8,21 +6,11 @@ import {assert} from './assert';
 export async function setupReplicache(): Promise<Replicache<M>> {
   const {pathname} = window.location;
   const paths = pathname.split('/');
-  let listId =
-    paths.indexOf('listId') > -1 ? paths[paths.indexOf('listId') + 1] : '';
-
-  if (listId) {
-    const data = await spaceExists(listId);
-    if (data.spaceExists === false) {
-      const spaceRes = await createSpace(listId);
-      listId = spaceRes.spaceID;
-    }
-  }
 
-  if (!listId) {
-    const spaceRes = await createSpace();
-    listId = spaceRes.spaceID;
-    window.location.href = '/listId/' + listId;
+  const spaceID =
+    paths.indexOf('listID') > -1 ? paths[paths.indexOf('listID') + 1] : '';
+  if (!spaceID || !(await spaceExists(spaceID))) {
+    window.location.href = '/listID/' + (await createSpace());
```

Comment:
The directory should be called `list` not `listID`:

<img width="257" alt="Screen Shot 2022-09-19 at 7 01 25 PM" src="https://user-images.githubusercontent.com/80388/191171460-6d60b5bc-e574-4e68-8627-1e4ed612034c.png">